test/spec/auth/connection-string.yml

```diff
@@ -287,6 +287,11 @@ tests:
         uri: "mongodb://localhost/foo"
         valid: true
         credential: ~
+    -
```

Comment:
was this the only test added as part of this work?

Reply:
I cherry-picked this spec, this is the only change. I removed the `.rst` files.